# States of each end effector magnet set. True means magnetic
bool magnet1
bool magnet2